src/main/java/org/jabref/logic/layout/LayoutEntry.java

```diff
@@ -360,7 +368,10 @@ public String doLayout(BibDatabaseContext databaseContext, Charset encoding) {
                 throw new UnsupportedOperationException("field and group ends not allowed in begin or end layout");
 
             case LayoutHelper.IS_OPTION_FIELD:
-                String field = BibDatabase.getText(text, databaseContext.getDatabase());
+                String field = Optional.ofNullable(databaseContext)
```

Comment:
I don't think this is necessary - the old code assumed that `databaseContext` is never null (otherwise an NPE would be there)